server/events/command_context.go

```diff
@@ -49,4 +50,7 @@ type CommandContext struct {
 	PullStatus *models.PullStatus
 
 	Trigger CommandTrigger
+
+	// Time Atlantis received VCS event, triggering command to be executed
+	EventTimestamp time.Time
```

Comment:
Technically commands shouldn't know about events so this naming seems odd, maybe like `TriggerTimestamp` or something like that?